libs/dao/src/main/java/com/akto/dao/testing/DeleteRunsInfoDao.java

```diff
@@ -0,0 +1,88 @@
+package com.akto.dao.testing;
+
+import java.util.List;
+import java.util.Map;
+
+import org.bson.conversions.Bson;
+import org.bson.types.ObjectId;
+
+import com.akto.dao.AccountsContextDao;
+import com.akto.dao.testing_run_findings.TestingRunIssuesDao;
+import com.akto.dto.test_run_findings.TestingRunIssues;
+import com.akto.dto.testing.DeleteRunsInfo;
+import com.akto.dto.testing.TestingRunResult;
+import com.mongodb.client.model.Filters;
+import com.mongodb.client.model.Updates;
+
+public class DeleteRunsInfoDao extends AccountsContextDao<DeleteRunsInfo> {
```

Comment:
DeleteRunsInfoDao is a bad name, rename this to something understandable

Reply:
is deleteTestRuns a good name?